tests/openqa/webui/test_results.pm

```diff
@@ -38,6 +38,12 @@ sub upload_autoinst_log {
 
 sub handle_notify_popup {
     return undef unless match_has_tag 'openqa-dont-notify-me';
+    for my $i (1 .. 5) {
+        assert_and_click 'openqa-dont-notify-me';
+        if (check_screen('openqa-tutorial-confirm')) {
```

Comment:
check screen also for `openqa-dont-notify-toggled`, and if so, click the confirm? https://openqa.opensuse.org/tests/1288011#step/test_results/7

Reply:
The confirm button is only visible when the checkbox is already toggled.

Reply:
Yeah, but in the test that I reference, the button is shown already :P, and openQA is looking for the unchecked box...

Reply:
Ah - seems that `check_screen` doesn't have any timeout which leads to thes race condition.
Adding a timeout should do it.